tests/fixtures.py

```diff
@@ -118,14 +117,48 @@ def __call__(self, csr):
 
 
 # "Correct" subject prefix for test data
-subject_prefix = (('O', 'Example inc.'), ('OU', 'Example Dept'),)
+subject_prefix = (
+    ("O", "Example inc."),
+    ("OU", "Example Dept"),
+)
+
+
+class VerbosityData(object):
```

Comment:
These are not test fixtures. They are validation or test-matrices.

Reply:
Indeed, but I was weighing adding additional files, which wouldnt necessarily help with readability and since the naming convention is [...]Data, I found it logically to add additional data there.